src/index.js

```diff
@@ -0,0 +1,636 @@
+        "RudderStack JavaScript SDK snippet included more than once.",
+      );
+    } else {
+      rudderanalytics.snippetExecuted = true;
+      window.rudderAnalyticsBuildType = "legacy";
+      var sdkBaseUrl = "https://cdn.rudderlabs.com/v3";
+      var sdkName = "rsa.min.js";
+      var scriptLoadingMode = "async";
+      var methods = [
+        "setDefaultInstanceKey",
+        "load",
+        "ready",
+        "page",
+        "track",
+        "identify",
+        "alias",
+        "group",
+        "reset",
+        "setAnonymousId",
+        "startSession",
+        "endSession",
+        "consent",
+      ];
+      methods.forEach((method) => {
+        rudderanalytics[method] = (function (methodName) {
+          return function () {
+            if (Array.isArray(window[identifier])) {
+              rudderanalytics.push(
+                [methodName].concat(Array.prototype.slice.call(arguments)),
+              );
+            } else {
+              var _methodName;
+              (_methodName = window[identifier][methodName]) === null ||
+                _methodName === void 0 ||
+                _methodName.apply(window[identifier], arguments);
+            }
+          };
+        })(method);
+      });
+      let supportsDynamicImport = false;
+      try {
+        window.rudderAnalyticsBuildType = "modern";
+      } catch (e) {}
+      var head = document.head || document.getElementsByTagName("head")[0];
+      var body = document.body || document.getElementsByTagName("body")[0];
+      window.rudderAnalyticsAddScript = function (
+        url,
+        extraAttributeKey,
+        extraAttributeVal,
+      ) {
+        var scriptTag = document.createElement("script");
+        scriptTag.src = url;
+        scriptTag.setAttribute("data-loader", "RS_JS_SDK");
+        if (extraAttributeKey && extraAttributeVal) {
+          scriptTag.setAttribute(extraAttributeKey, extraAttributeVal);
+        }
+        if (scriptLoadingMode === "async") {
+          scriptTag.async = true;
+        } else if (scriptLoadingMode === "defer") {
+          scriptTag.defer = true;
+        }
+        if (head) {
+          head.insertBefore(scriptTag, head.firstChild);
+        } else {
+          body.insertBefore(scriptTag, body.firstChild);
+        }
+      };
+      window.rudderAnalyticsMount = function () {
+        if (typeof globalThis === "undefined") {
+          Object.defineProperty(Object.prototype, "__globalThis_magic__", {
+            get: function get() {
+              return this;
+            },
+            configurable: true,
+          });
+          __globalThis_magic__.globalThis = __globalThis_magic__;
+          delete Object.prototype.__globalThis_magic__;
+        }
+        window.rudderAnalyticsAddScript(
+          ""
+            .concat(sdkBaseUrl, "/")
+            .concat(window.rudderAnalyticsBuildType, "/")
+            .concat(sdkName),
+          "data-rsa-write-key",
+          "<WRITE_KEY>",
```

Comment:
We can reference the variable directly.
```suggestion
          "WRITE_KEY",
```